src/ResourceManager/Sql/Commands.Sql/Security/Cmdlet/SqlCmdletBase.cs

```diff
@@ -37,47 +41,57 @@ public abstract class SqlDatabaseSecurityCmdletBase : SqlDatabaseCmdletBase
         [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The name of the resource group")]
         [ValidateNotNullOrEmpty]
         public string ResourceGroupName { get; set; }
-
+        
         /// <summary>
+        /// Gets or sets the name of the database server to use.
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "SQL Database server name.")]
```

Comment:
This should not be a mandatory parameter if it is in the SqlCmdletBase class.  For example Get-AzureSqlDatabaseServer does not require a server name; in which case it would return all servers in the resource group.